controllers/reconcile.go

```diff
@@ -463,18 +463,22 @@ func (r *FrontendReconciliation) populatePushCacheContainer(j *batchv1.Job) erro
 	j.Spec.Template.Spec.Volumes = []v1.Volume{pushCacheVolume}
 
 	secrets := v1.SecretList{}
-	r.Client.List(r.Ctx, &secrets, client.InNamespace(r.Frontend.Namespace))
+	err := r.Client.List(r.Ctx, &secrets, client.InNamespace(r.Frontend.Namespace))
+	if err != nil {
+		return err
+	}
 
 	objectStoreInfo, err := ExtractBucketConfigFromSecret(secrets.Items, r.FrontendEnvironment.Spec.PushCacheBucket)
 	if err != nil {
-
+		return err
 	}
-	aws_username := objectStoreInfo.AccessKey
-	aws_password := objectStoreInfo.SecretKey
+
+	awsUsername := objectStoreInfo.AccessKey
+	awsPassword := objectStoreInfo.SecretKey
 
 	// TODO: Is the source directory "-s $(pwd) accurate"??
 	// Construct the pushcache startup command
-	command := fmt.Sprintf("sleep 120; valpop populate -r data/${ROUTE_PATH} -s $(pwd) --hostname 127.0.0.1 --username ${%s} --password ${%s}", *aws_username, *aws_password)
+	command := fmt.Sprintf("sleep 120; valpop populate -r data/${ROUTE_PATH} -s ${DIST_FOLDER} --hostname 127.0.0.1 --username ${%s} --password ${%s}", *awsUsername, *awsPassword)
```

Comment:
I think we might be still missing the hostname. I don't think `127.0.0.1` will do the job.